tests/integration/fixtures/test_connect.py

```diff
@@ -1,3 +1,8 @@
 def test_databricks_connect(spark):
     rows = spark.sql("SELECT 1").collect()
     assert rows[0][0] == 1
+
+
+def test_databricks_connect_serverless(serverless_env, spark):
```

Comment:
Can we test somehow that this really running on serverless? For example, get a spark configuration parameter that is only valid on serverless?

Reply:
yes, will add this

Reply:
added, the only way I found is to check if the cluster has a creator. Serverless clusters don't have creator assigned. There are no useful tags or anything similar that we could use.